samples/build.gradle

```diff
@@ -21,6 +21,18 @@
 
 apply from: "$buildScriptsDir/common-java.gradle"
 
+import com.microsoft.applicationinsights.build.tasks.PropsFileGen
+
+def sourceOriginFileDir = "$project.buildDir/src/main/resources"
```

Comment:
To make this sample stand alone please do not use the `PropsFileGen` task.
Instead change this task to create the properties file